pkg/monitortests/kubeapiserver/incluster_disruption_serializer/remote.go

```diff
@@ -246,9 +157,11 @@ func createServiceNetworkDS(ctx context.Context, clientset *kubernetes.Clientset
 	return nil
 }
 
-func createLocalhostDS(ctx context.Context, clientset *kubernetes.Clientset) error {
+func createLocalhostDS(ctx context.Context, clientset kubernetes.Interface, openshiftTestsImagePullSpec string) error {
 	dsObj := resourceread.ReadDaemonSetV1OrDie(dsLocalhostYaml)
 	dsObj.Namespace = namespace
+	// we need to use the openshift-tests image of the destination during an upgrade.
+	dsObj.Spec.Template.Spec.Containers[0].Image = openshiftTestsImagePullSpec
```

Comment:
found this while going through.  I'm guessing this is why you had bash, but I think this is what you needed: openshift-tests image must be the latest.